### ISSUE TITLE
From which class to inherit data objects?

### ISSUE THREAD

**Issue by gtennis** [evermeer/EVCloudKitDao#80]
First, awesome project, thanks a lot for making it open source!

Now, back to my problem. It's written in readme we should inherit from EVCloudData, however, it gives an error "... cannot be constructed because it has no accessible constructors when using "connectToNews() from the example and providing News() as param". 

However, in the AppMessage example project, News entity inherit from CKDataObject. Any ideas what am I doing wrong?

**Comment by evermeer** [comment]
Ah, I should have updated the readme file. Some time ago I have moved some code to a subspec of EVReflection. So now instead of having a dependency to EVReflection, there is a dependency to EVReflection/CloudKit. While doing that I also renamed the EVCloudData object to CKData object.

The reason for this move was the request from developers that they loved the easy object mapping but wanted to have full control over handeling all functions. Instead of creating a separate cocoapod for that I moved it to EVReflection so that I would also always run all unit tests when updating EVReflection. Since EVReflection is much more popular (200K downloads) it's also maintained better. This will make sure that the CloudKit subspec keeps on working when making changes.
